esp/src/eclwatch/ShowIndividualPermissionsWidget.js

```diff
@@ -1,40 +1,74 @@
 define([
     "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/query",
     "src/nlsHPCC",
+    "dojo/dom-form",
 
     "dijit/registry",
     "dijit/form/CheckBox",
 
     "dgrid/editor",
+    "dgrid/selector",
 
-    "hpcc/GridDetailsWidget",
+    "hpcc/_TabContainerWidget",
     "src/ws_access",
-    "src/ESPUtil"
+    "src/ESPUtil",
 
-], function (declare, nlsHPCCMod,
+    "dojo/text!../templates/ShowIndividualPermissionsWidget.html",
+
+    "dijit/layout/BorderContainer",
+    "dijit/layout/TabContainer",
+    "dijit/layout/ContentPane",
+    "dijit/Toolbar",
+    "dijit/form/Form",
+    "dijit/form/Button",
+    "dijit/form/ToggleButton",
+    "dijit/form/DropDownButton",
+    "dijit/form/ValidationTextBox",
+    "dijit/ToolbarSeparator",
+    "dijit/form/TextBox",
+    "dijit/Dialog",
+    "dijit/TooltipDialog",
+
+    "hpcc/TableContainer"
+
+], function (declare, lang, query, nlsHPCCMod, domForm,
     registry, CheckBox,
-    editor,
-    GridDetailsWidget, WsAccess, ESPUtil) {
+    editor, selector,
+    _TabContainerWidget, WsAccess, ESPUtil, template) {
 
     var nlsHPCC = nlsHPCCMod.default;
-    return declare("ShowIndividualPermissionsWidget", [GridDetailsWidget], {
+    return declare("ShowIndividualPermissionsWidget", [_TabContainerWidget], {
+        templateString: template,
+        baseClass: "ShowIndividualPermissionsWidget",
         i18n: nlsHPCC,
 
         gridTitle: nlsHPCC.title_Permissions,
         idProperty: "__hpcc_id",
 
+        postCreate: function (args) {
+            this.inherited(arguments);
+            this.addGroupForm = registry.byId(this.id + "AddGroupForm");
+        },
+
         //  Hitched Actions  ---
         _onRefresh: function (args) {
             this.grid.refresh();
         },
 
+        fixLabels: function () {
+            query(".AddGroupLabel", this.id + "AddGroupForm").forEach(function (node) {
+                console.log(node);
```

Comment:
Left over debugging?

Reply:
I forgot to revisit, wanted to fix those labels which contained HTML